packages/main/src/Select.ts

```diff
@@ -371,9 +351,6 @@ class Select extends UI5Element implements IFormElement {
 	 *
 	 * <br><br>
 	 * <b>Note:</b> Use the <code>ui5-option</code> component to define the desired options.
-	 * @type {sap.ui.webc.main.ISelectOption[]}
-	 * @slot options
-	 * @name sap.ui.webc.main.Select.prototype.default
 	 * @public
 	 */
 	@slot({ "default": true, type: HTMLElement, invalidateOnChildChange: true })
```

Comment:
Here we have to use ISelectOption as a type of the slot. If the interface doesn't exist create it as it's described.